salt/runners/lxc.py

```diff
@@ -251,7 +251,7 @@ def init(names, host=None, saltcloud_mode=False, quiet=False, **kwargs):
                 expr_form='list', timeout=600).get(host, {})
         name = kw.pop('name', name)
         # be sure not to seed an alrady seeded host
-        kw['seed'] = seeds[name]
+        kw['seed'] = seeds.get(name, True)
```

Comment:
Oh, sorry, missed that. kwargs.get('seed', True) then. I'll prepare new pull request.